### PR TITLE
chore: update bundle name

### PR DESCRIPTION
**Pull request by LinkW77**
# Pull Request

## PR Description

Update bundle name

## Type of Change

- [ ] Bug fix
- [ ] New feature
- [ ] Performance enhancement
- [ ] Code refactor
- [x] Documentation update
- [ ] Other, please describe:

## Checklist

Before submitting this PR, please make sure:

- [x] I have read the [CONTRIBUTING.md](../CONTRIBUTING.md) guidelines.
- [x] I have tested my changes locally to ensure they are effective.
- [x] I have updated the necessary documentation (if applicable).

### CODE REVIEW

**Review by jameszyao — DISMISSED** [review]
LGTM

File: plugin/bundles/serper/resources/i18n/en.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-bundle_name: "Serper search"
+bundle_name: "Serper"
```
Comment (jameszyao) [inline plugin/bundles/serper/resources/i18n/en.yml:1]: Don't change

File: plugin/bundles/stability_ai/resources/i18n/en.yml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-bundle_name: "Stability AI - Image Generation"
+bundle_name: "Stability AI"
```
Comment (jameszyao) [inline plugin/bundles/stability_ai/resources/i18n/en.yml:1]: Don't change

File: plugin/bundles/arxiv_search/resources/i18n/en.yml [hunk 3]
```diff
@@ -1,4 +1,4 @@
-bundle_name: "arXiv Search"
+bundle_name: "arXiv search"
```
Comment (jameszyao) [inline plugin/bundles/arxiv_search/resources/i18n/en.yml:1]: Don't change

File: plugin/bundles/api_ninjas_commodity_price/resources/i18n/en.yml [hunk 4]
```diff
@@ -1,4 +1,4 @@
-bundle_name: "API Ninjas Commodity Price"
+bundle_name: "Commodity Price API"
```
Comment (jameszyao) [inline plugin/bundles/api_ninjas_commodity_price/resources/i18n/en.yml:1]: Don't change

**Review by DynamesC — APPROVED** [review]
LGTM

**Review by jameszyao — APPROVED** [review]
LGTM

**Pull request merged**